lib/cpu.js

```diff
@@ -83,17 +84,19 @@ function getCpu() {
         });
       }
       if (_linux) {
-        exec("cat /proc/cpuinfo | grep 'model name'", function (error, stdout) {
+        exec("cat /proc/cpuinfo | grep -E 'model name|Revision'", function (error, stdout) {
```

Comment:
The entire file could be parsed more effectively and returned. As it stands, I have a need for this specific feature and would like to continue using this library.